lib/workers/repository/update/pr/index.ts

```diff
@@ -413,7 +413,7 @@ export async function ensurePr(
       // istanbul ignore if
       if (GlobalConfig.get('dryRun')) {
         logger.info(`DRY-RUN: Would update PR #${existingPr.number}`);
-      } else {
+      } else if (!GlobalConfig.get('dryRun')) {
```

Comment:
I think these added if statements in this file can be reverted? There's an if before them that checks if dryRun is set so these can't possibly be false